src/translib/transformer/xlate_utils.go

```diff
@@ -657,7 +697,7 @@ func unmarshalJsonToDbData(schema *yang.Entry, fieldName string, value interface
         switch ykind {
         case yang.Ystring, yang.Ydecimal64, yang.Yint64, yang.Yuint64:
```

Comment:
Is this intended?? Go executes only selected case block. It does not run following case block in the absence of break statement. So, case statements at line 698 and 702 are no-op blocks. They wont set variable **data**. Function is going to return empty string.

We are seeing multiple issues with transformer not updating fields in db. I think this is the root cause.

Reply:
Below fix should resolve the issue.

Reply:
Ok.. can you please add comment at line 698 and 702 that they are no-op cases? Otherwise it looks like coding error.